Makefile

```diff
@@ -46,7 +46,7 @@ tox: venv
 	tox
 
 flake:
-	flake8 hy tests --ignore=E121,E123,E126,E226,E24,E704,W503,E305
+	flake8 hy tests --ignore=E121,E123,E126,E226,E24,E704,W503,E302,E305,E701
```

Comment:
If these new error ignores are solely for the single-line `def`s and close-together `with`s in the tests, maybe it'd be worth it running `flake8` on just that file separately, or using `# noqa` comments (which is what I think would be better).